polynote-server/src/main/scala/polynote/server/Server.scala

```diff
@@ -81,27 +78,12 @@ class Server {
       |before running Polynote. You are solely responsible for any breach, loss, or damage caused by running
       |this software insecurely.""".stripMargin
 
-  private val banner: String =
```

Comment:
😢

Reply:
Replaced it with escaped strings that don't break IntelliJ